src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -118,6 +118,18 @@
       + "provide fixed width filenames that can be ordered by simple lexicographic sorting.";
   public static final int FILENAME_OFFSET_ZERO_PAD_WIDTH_DEFAULT = 10;
 
+  public static final String SECURE_HADOOP_CONFIG = "is.secure.hadoop";
+  private static final String SECURE_HADOOP_DOC = "Config indicating whether the hadoop cluster is secure or not";
+  private static boolean SECURE_HADOOP_DEFAULT = false;
+
+  public static final String HDFS_CONNECTOR_PRINCIPAL_CONFIG = "hdfs.connector.principal";
+  private static final String HDFS_CONNECTOR_PRINCIPAL_DOC =
+      "The principal to use when authenticating the connector with a secure hdfs cluster.";
```

Comment:
Does the principal need to match an actual OS user? (It does if you create "local" temp files, like MR does). 
If we need it, please document the requirement.